internal/configuration/validator/const.go

```diff
@@ -130,8 +129,9 @@ var validKeys = []string{
 }
 
 var specificErrorKeys = map[string]string{
-	"logs_file_path": "config key replaced: logs_file is now log_file",
-	"logs_level":     "config key replaced: logs_level is now log_level",
+	"logs_file_path":   "config key replaced: logs_file is now log_file",
+	"logs_level":       "config key replaced: logs_level is now log_level",
+	"google_analytics": "config key incorrect: google_analytics - this functionality has been removed",
```

Comment:
```suggestion
	"google_analytics": "config key removed: google_analytics - this functionality has been deprecated",
```

I feel like suggesting the key is “incorrect” isn’t actually appropriate language.